tests/runner/spec.js

```diff
@@ -359,6 +359,11 @@ describe('Compile', function(){
       assert.equal(' abd', render(vm, {foo: 'a', bar: 'b', dd: 'd'}))
     })
 
+    it('#macro map argument', function(){
+      var vm = '#macro( d $a)#foreach($_item in $a.entrySet())$_item.key = $_item.value #end#end #d ( {"foo": $foo,"bar":$bar} )'
+      assert.equal(' foo = a bar = b ', render(vm, {foo: 'a', bar: 'b'}))
+    })
+
```

Comment:
This spec file just run on browser. But for velocity.js, there is no anything about dom, so nodejs is just the same as browser.

So, this file is no necessary.